### ISSUE TITLE
LaTeX doc is broken

### ISSUE THREAD

**Issue by benediktahrens** [UniMath/UniMath#656]
Fails with
```
(/usr/share/texlive/texmf-dist/tex/latex/ucs/data/uni-3.def) [3]
! Missing $ inserted.
<inserted text> 
                $
l.207 ... match w as t0 return TYPE with | tpair _
                                                   _ x y => BODY end
? 
```
One solution would be to put that in verbatim mode, I think:

```
<<<
  put verbatim stuff here
>>>
```

**Comment by DanGrayson** [comment]
I just tried putting them inside `[ .... ]` and that didn't work ...

**Comment by DanGrayson** [comment]
Using `<<< ... >>>` didn't work either.

Nor did 
```
\begin{verbatim}
       match w as t0 return TYPE with | tpair _ _ x y => BODY end
\end{verbatim}
```

**Comment by benediktahrens** [comment]
On 03/16/2017 04:03 PM, Daniel R. Grayson wrote:
> Using `<<< ... >>>` didn't work either.
> 
> Nor did 
> ```
> \begin{verbatim}
>        match w as t0 return TYPE with | tpair _ _ x y => BODY end
> \end{verbatim}
> ```

Mmh, I don't have any idea other than what you have tried already...

**Comment by benediktahrens** [comment]
Somebody had put a `%` into the comment, this messed everything afterwards up.
A fix is on its way...

**Comment by benediktahrens** [comment]
Resolved by #697 .
